### PR TITLE
GP2-1398 Data loading performance

### PR DESCRIPTION
**Pull request by bobmeredith**
This PR implements blocked requests for comtrade and country data
Rather than use country name as previous requests, the new helpers/api fns take a list of country ISO2 codes and return data collated by those codes.
The country_data request also accepts a list of fields to retrieve where the fieldname is the model name in directory-api. Again the results are assembled into a dict, mapped by field/model name

### Workflow

- [x] Ticket exists in Jira https://uktrade.atlassian.net/browse/GP2-1398
- [x] Jira ticket has the correct status.
- [x] [Changelog](CHANGELOG.md) entry added.

### Reviewing help

- [x] Explains how to test locally, including how to set up appropriate data

### Housekeeping

- [x] Python requirements have been re-compiled.
- [x] Frontend assets have been re-compiled.

### Merging

- [x] This PR can be merged by reviewers. (If unticked, please leave for the author to merge)

### CODE REVIEW

**Comment by bobmeredith** [comment]
Testing: This PR should massively impact the loading speed of the economy tab of the 'Where to export' page.
All data validation tests applicable to this page should be repeated, since the data and the calculation for change percentage have been changed.
In particular, observe behaviour for countries with missing data (UK or world) and with only data for previous years.

File: react-components/src/components/CompareMarkets/TabConfigEconomy.jsx [hunk 1]
```diff
@@ -11,7 +11,7 @@ const rankOutOf = (data, key) => {
 }
 
 const sign = (value) => {
```
Comment (dahfool) [inline react-components/src/components/CompareMarkets/TabConfigEconomy.jsx:13]: Is this function appending + or - to a number ? if yes i think it should be move into utils and reuse in Export plan.

Comment (bobmeredith) [reply react-components/src/components/CompareMarkets/TabConfigEconomy.jsx:13]: Yes Good call.

**Review by webbyfox — APPROVED** [review]
LGTM @bobmeredith

**Pull request merged**